Resources/Textures/Shaders/drowsiness.swsl

```diff
@@ -0,0 +1,17 @@
+﻿uniform sampler2D SCREEN_TEXTURE;
+uniform highp float VisualScale; // between 0 and 1
+const highp float Intensity = 0.2;
+const highp int SampleCount = 8; // multiple of 2
+
+// a simple radial blur
```

Comment:
would prefer to see this file just called `radial-blur.swsl` or something then, seems reusable

Reply:
done